weddineo-frontend/apps/weddineo-frontend/src/app/shared/interceptor/token.interceptor.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  HttpEvent,
+  HttpHandler,
+  HttpInterceptor,
+  HttpRequest,
+} from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class TokenInterceptor implements HttpInterceptor {
+  intercept(
+    req: HttpRequest<any>,
+    next: HttpHandler
+  ): Observable<HttpEvent<any>> {
+    const token = localStorage.getItem('ID_TOKEN');
```

Comment:
If I remeber correctly, storing tokens in localstorage i single page applictions is a security issue

Reply:
You can get it from every request from header. But you have right, it will be stored in store in the future.